README.md

```diff
@@ -155,9 +155,9 @@ for details.
 
 - [connect-kotlin][connect-kotlin]: Idiomatic gRPC & Connect RPCs for Kotlin
 - [connect-go][connect-go]: Go service stubs for servers
-- [connect-web][connect-web]: TypeScript clients for web browsers
+- [connect-es][connect-es]: Type-safe APIs with Protobuf and TypeScript
 - [Buf Studio][buf-studio]: Web UI for ad-hoc RPCs
-- [connect-crosstest][connect-crosstest]: Connect, gRPC, and gRPC-Web
+- [conformance][connect-conformance]: Connect, gRPC, and gRPC-Web
```

Comment:
```suggestion
- [connect-conformance][connect-conformance]: Connect, gRPC, and gRPC-Web
```

Reply:
It was actually decided to use just `conformance` for this link (see [here](https://github.com/bufbuild/connect-crosstest/pull/539#discussion_r1313260511) and [here](https://github.com/connectrpc/connect-go/pull/579#discussion_r1313672056)). But I can make the link reference also be `conformance` for consistency if you like?

Reply:
I don't have a super strong opinion, good to keep it consistent across repos